### PR TITLE
Add sstable upgrade support to the node

### PR DESCRIPTION
**Pull request by juliayakovlev**

### CODE REVIEW

File: ccmlib/node.py [hunk 1]
```diff
@@ -322,6 +322,39 @@ def logfilename(self):
...
+        stdout, _ = self.nodetool('help')
+        return True if "upgradesstables" in stdout else False
+
+    def get_node_supported_sstable_versions(self):
+        match = self.grep_log(r'Feature (.*)_SSTABLE_FORMAT is enabled')
+        return [m[1].group(1).lower() for m in match] if match else []
+
+    def check_node_sstables_format(self, timeout=10):
+        node_system_folder = os.path.join(self.get_path(), 'data', 'system')
+        find_cmd = f"find {node_system_folder} -type f ! -path '*snapshots*' -printf %f\\n".split()
```
Comment (amoskong) [inline ccmlib/node.py:335]: I think the find cmdline is faster than `os.walk() + some additional filter`.
So it's fine.

File: ccmlib/node.py [hunk 2]
```diff
@@ -322,6 +322,39 @@ def logfilename(self):
...
+
+            sstable_version_regex = re.compile(r'(\w+)-\d+-(.+)\.(db|txt|sha1|crc32)')
+
+            sstable_versions = {sstable_version_regex.search(f).group(1) for f in all_sstable_files if
+                                sstable_version_regex.search(f)}
+            return sstable_versions
+        except Exception:
+            raise
+
+    def run(self, cmd, timeout):
```
Comment (amoskong) [inline ccmlib/node.py:350]: Can you move this to a ccmlib/common.py? The helper is very general.

Comment (juliayakovlev) [reply ccmlib/node.py]: Done

**Review by amoskong — APPROVED** [review]
LGTM

**Comment by amoskong** [comment]
@scylladb/ccm-maint  @roydahan can you help to review & merge this PR?

File: ccmlib/node.py [hunk 3]
```diff
@@ -322,6 +323,31 @@ def logfilename(self):
...
+
+    def get_node_supported_sstable_versions(self):
+        match = self.grep_log(r'Feature (.*)_SSTABLE_FORMAT is enabled')
+        return [m[1].group(1).lower() for m in match] if match else []
+
+    def check_node_sstables_format(self, timeout=10):
+        node_system_folder = os.path.join(self.get_path(), 'data', 'system')
+        find_cmd = f"find {node_system_folder} -type f ! -path '*snapshots*' -printf %f\\n".split()
+        try:
+            stdout, stderr = run(find_cmd, timeout)
```
Comment (bentsi) [inline ccmlib/node.py:338]: use `subprocess.run()`

**Review by bentsi — CHANGES_REQUESTED** [review]
address a small comment

File: ccmlib/node.py (hunk 3 above)
Comment (juliayakovlev) [reply ccmlib/node.py]: Fixed

**Comment by juliayakovlev** [comment]
Test passed.
Test result: https://jenkins.scylladb.com/view/master/job/scylla-master/job/byo/job/dtest-byo/411/testReport/

@bentsi @roydahan @scylladb/ccm-maint 
Please, review

**Review by bentsi — APPROVED** [review]

**Pull request merged**